### PR TITLE
parity_netPeers function

### PR DESCRIPTION
**Pull request by blanksteer**
geth returns a different structure. believe it is of type HashMap. thought this would be a simpler/better alternative atm until i have time to do some digging through geth's source. will setup a seperate func for geth. hope the naming scheme is ok. ``func parity_peer_list`` and ``func geth_peer_list``.

### CODE REVIEW

File: src/api/net.rs [hunk 1]
```diff
@@ -36,6 +36,11 @@ impl<T: Transport> Net<T> {
         CallFuture::new(self.transport.execute("net_peerCount", vec![]))
     }
 
+    /// Returns list of connected nodes parity
+    pub fn parity_peer_list(&self) -> CallFuture<PeerType, T::Out>{
```
Comment (tomusdrw) [inline src/api/net.rs]: Let's move this to `parity_` namespace then

File: src/types/node.rs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+//! Type of node (Parity, Geth...)
+
+/// specify behavior of node interacting with
+pub enum NodeType {
```
Comment (tomusdrw) [inline src/types/node.rs]: This doesn't seem required now

File: src/types/peer.rs [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+/// ip address of both local and remote
+#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all="camelCase")]
+pub struct PeerNetworkInfo {
+    pub remote_address: String,
+    pub local_address: String,
+}
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct PeerProtocolsInfo {
```
Comment (tomusdrw) [inline src/types/peer.rs]: More docs

File: src/api/net.rs (hunk 1 above)
Comment (blanksteer) [reply src/api/net.rs]: will do! :) I think it's ``ParitySet``? Didn't realize you gave these instructions. Asked if i had the go ahead on [#198](https://github.com/tomusdrw/rust-web3/pull/198)

Comment (tomusdrw) [reply src/api/net.rs]: Sounds good!

**Pull request closed without merging**